recipes/mongo-c-driver/all/conanfile.py

```diff
@@ -0,0 +1,155 @@
+        # todo: apply patch https://github.com/microsoft/vcpkg/blob/master/ports/mongo-c-driver/disable-static-when-dynamic-build.patch
+        self._cmake.definitions["ENABLE_STATIC"] = "OFF" if self.options.shared else "ON"
+        self._cmake.definitions["ENABLE_SSL"] = self.options.ssl_provider
+        self._cmake.definitions["ENABLE_ZLIB"] = self.options.zlib
+        self._cmake.definitions["ENABLE_SHM_COUNTERS"] = "OFF"
+        if self.options.zlib == 'SYSTEM':
+            self._cmake.definitions["ZLIB_ROOT"] = self.deps_cpp_info["zlib"].rootpath
+        if self.options.ssl_provider == 'OPENSSL':
+            self._cmake.definitions["OPENSSL_ROOT_DIR"] = self.deps_cpp_info["openssl"].rootpath
+
+        self._cmake.configure(
+            source_folder=self._source_subfolder,
+            build_folder=self._build_subfolder
+        )
+
+        return self._cmake
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "{name}-{version}".format(
+            name=self.name,
+            version=self.version
+        )
+        os.rename(extracted_dir, self._source_subfolder)
+        # todo: fix it https://github.com/bincrafters/community/issues/995
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "src", "libmongoc", "CMakeLists.txt"),
+            "add_executable (mongoc-stat ${PROJECT_SOURCE_DIR}/../../src/tools/mongoc-stat.c)",
+            "# add_executable (mongoc-stat ${PROJECT_SOURCE_DIR}/../../src/tools/mongoc-stat.c)")
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "src", "libmongoc", "CMakeLists.txt"),
+            "target_link_libraries (mongoc-stat mongoc_shared ${LIBRARIES})",
+            "# target_link_libraries (mongoc-stat mongoc_shared ${LIBRARIES})")
+        if self.options.ssl_provider == 'LIBRESSL':
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "src", "libmongoc", "CMakeLists.txt"),
+                self._libressl_find_pattern,
+                self._libressl_replacement_pattern.format(
+                    LIBRESSL_INCLUDE_DIRS=self.deps_cpp_info["libressl"].include_paths[0],
+                    LIBRESSL_LIBRARY_DIRS=self.deps_cpp_info["libressl"].lib_paths[0]
+                ))
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.options.shared:
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        if self.options.ssl_provider == 'OPENSSL':
+            self.requires("openssl/1.1.1g")
+        if self.options.ssl_provider == 'LIBRESSL':
+            self.output.warn("Can be broken. Prefer OpenSSL instead of LIBRESSL")
```

Comment:
Again, can you please explain where this comes from?

Reply:
<details>
  <summary> I have linking problem on my macos (click to expand)</summary>
  
(venv) ➜  all git:(mongo-c-driver) ✗ conan create . mongo-c-driver/1.17.2@dev/testing --build
Exporting package recipe
mongo-c-driver/1.17.2@dev/testing exports: File 'conandata.yml' found. Exporting it...
mongo-c-driver/1.17.2@dev/testing exports: Copied 1 '.yml' file: conandata.yml
mongo-c-driver/1.17.2@dev/testing: A new conanfile.py version was exported
mongo-c-driver/1.17.2@dev/testing: Folder: /Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/export
mongo-c-driver/1.17.2@dev/testing: Exported revision: 8a131cf57412df58aeb2d8a656dd0b2b
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=apple-clang
compiler.libcxx=libc++
compiler.version=11.0
os=Macos
os_build=Macos
[options]
[build_requires]
[env]

mongo-c-driver/1.17.2@dev/testing: WARN: Can be broken. Prefer OpenSSL instead of LIBRESSL
libressl/3.2.0: Forced build from source
mongo-c-driver/1.17.2@dev/testing: Forced build from source
mongo-c-driver/1.17.2@dev/testing (test package): Installing package
Requirements
    libressl/3.2.0 from 'conan-center' - Cache
    mongo-c-driver/1.17.2@dev/testing from local cache - Cache
Packages
    libressl/3.2.0:f74366f76f700cc6e991285892ad7a23c30e6d47 - Build
    mongo-c-driver/1.17.2@dev/testing:f03349b8a70d68ed7d068c77dae611fdcfdbcf9e - Build

Installing (downloading, building) binaries...
libressl/3.2.0: Copying sources to build folder
libressl/3.2.0: Building your package in /Users/n1/.conan/data/libressl/3.2.0/_/_/build/f74366f76f700cc6e991285892ad7a23c30e6d47
libressl/3.2.0: Generator cmake created conanbuildinfo.cmake
libressl/3.2.0: Calling build()
-- The C compiler identification is AppleClang 11.0.0.11000033
-- The CXX compiler identification is AppleClang 11.0.0.11000033
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- The ASM compiler identification is Clang
-- Found assembler: /Library/Developer/CommandLineTools/usr/bin/cc
-- Looking for asprintf
-- Looking for asprintf - found
-- Looking for reallocarray
-- Looking for reallocarray - not found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strlcat
-- Looking for strlcat - found
-- Looking for strlcpy
-- Looking for strlcpy - found
-- Looking for strndup
-- Looking for strndup - found
-- Looking for strnlen
-- Looking for strnlen - found
-- Looking for strsep
-- Looking for strsep - found
-- Looking for timegm
-- Looking for timegm - found
-- Looking for arc4random_buf
-- Looking for arc4random_buf - found
-- Looking for arc4random_uniform
-- Looking for arc4random_uniform - found
-- Looking for explicit_bzero
-- Looking for explicit_bzero - not found
-- Looking for getauxval
-- Looking for getauxval - not found
-- Looking for getentropy
-- Looking for getentropy - found
-- Looking for getpagesize
-- Looking for getpagesize - found
-- Looking for getprogname
-- Looking for getprogname - found
-- Looking for syslog_r
-- Looking for syslog_r - not found
-- Looking for syslog
-- Looking for syslog - found
-- Looking for timespecsub
-- Looking for timespecsub - not found
-- Looking for timingsafe_bcmp
-- Looking for timingsafe_bcmp - found
-- Looking for timingsafe_memcmp
-- Looking for timingsafe_memcmp - not found
-- Looking for memmem
-- Looking for memmem - found
-- Looking for include file err.h
-- Looking for include file err.h - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of time_t
-- Check size of time_t - done
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_SHARED
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    USE_SHARED


-- Build files have been written to: /Users/n1/.conan/data/libressl/3.2.0/_/_/build/f74366f76f700cc6e991285892ad7a23c30e6d47
Scanning dependencies of target crypto
[  0%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes-macosx-x86_64.S.o
[  0%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/bsaes-macosx-x86_64.S.o
[  0%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/vpaes-macosx-x86_64.S.o
[  0%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aesni-macosx-x86_64.S.o
[  0%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aesni-sha1-macosx-x86_64.S.o
[  0%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/modexp512-macosx-x86_64.S.o
[  1%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/mont-macosx-x86_64.S.o
[  1%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/mont5-macosx-x86_64.S.o
[  1%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/gf2m-macosx-x86_64.S.o
[  1%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/md5/md5-macosx-x86_64.S.o
[  1%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/camellia/cmll-macosx-x86_64.S.o
[  1%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/ghash-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc4/rc4-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc4/rc4-md5-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha1-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha256-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/whrlpool/wp-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha512-macosx-x86_64.S.o
[  2%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cpuid-macosx-x86_64.S.o
[  3%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cpt_err.c.o
[  3%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cryptlib.c.o
[  3%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/crypto_init.c.o
[  3%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cversion.c.o
[  3%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ex_data.c.o
[  3%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/malloc-wrapper.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/mem_clr.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/mem_dbg.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/o_init.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/o_str.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/o_time.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_cfb.c.o
[  4%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_ctr.c.o
[  5%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_ecb.c.o
[  5%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_misc.c.o
[  5%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_ige.c.o
[  5%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_ofb.c.o
[  5%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/aes/aes_wrap.c.o
[  5%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_bitstr.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_bool.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_d2i_fp.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_digest.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_dup.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_enum.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_i2d_fp.c.o
[  6%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_int.c.o
[  7%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_mbstr.c.o
[  7%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_object.c.o
[  7%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_octet.c.o
[  7%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_print.c.o
[  7%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_sign.c.o
[  7%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_strex.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_strnid.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_time.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_time_tm.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_type.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_utf8.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/a_verify.c.o
[  8%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/ameth_lib.c.o
[  9%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn1_err.c.o
[  9%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn1_gen.c.o
[  9%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn1_lib.c.o
[  9%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn1_par.c.o
[  9%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn_mime.c.o
[  9%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn_moid.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/asn_pack.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/bio_asn1.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/bio_ndef.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/d2i_pr.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/d2i_pu.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/evp_asn1.c.o
[ 10%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/f_enum.c.o
[ 11%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/f_int.c.o
[ 11%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/f_string.c.o
[ 11%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/i2d_pr.c.o
[ 11%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/i2d_pu.c.o
[ 11%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/n_pkey.c.o
[ 11%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/nsseq.c.o
[ 12%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/p5_pbe.c.o
[ 12%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/p5_pbev2.c.o
[ 12%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/p8_pkey.c.o
[ 12%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_bitst.c.o
[ 12%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_crl.c.o
[ 12%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_pkey.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_req.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_spki.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_x509.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/t_x509a.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_dec.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_enc.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_fre.c.o
[ 13%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_prn.c.o
[ 14%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_new.c.o
[ 14%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_typ.c.o
[ 14%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/tasn_utl.c.o
[ 14%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_algor.c.o
[ 14%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_attrib.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_bignum.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_crl.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_exten.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_info.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_long.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_name.c.o
[ 15%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_nx509.c.o
[ 16%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_pkey.c.o
[ 16%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_req.c.o
[ 16%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_pubkey.c.o
[ 16%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_sig.c.o
[ 16%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_spki.c.o
[ 16%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_val.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_x509.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/asn1/x_x509a.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bf/bf_ecb.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bf/bf_cfb64.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bf/bf_enc.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bf/bf_ofb64.c.o
[ 17%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bf/bf_skey.c.o
[ 18%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/b_dump.c.o
[ 18%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/b_print.c.o
[ 18%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/b_sock.c.o
[ 18%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bf_buff.c.o
[ 18%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bf_nbio.c.o
[ 18%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bf_null.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bio_cb.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bio_err.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bio_lib.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bio_meth.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_acpt.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_bio.c.o
[ 19%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_conn.c.o
[ 20%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_dgram.c.o
[ 20%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_fd.c.o
[ 20%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_file.c.o
[ 20%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_mem.c.o
[ 20%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_null.c.o
[ 20%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_sock.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_asm.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_add.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_blind.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_const.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_ctx.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_depr.c.o
[ 21%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_div.c.o
[ 22%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_err.c.o
[ 22%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_exp.c.o
[ 22%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_exp2.c.o
[ 22%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_gcd.c.o
[ 22%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_gf2m.c.o
[ 22%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_kron.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_lib.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_mod.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_mont.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_mpi.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_mul.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_nist.c.o
[ 23%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_prime.c.o
[ 24%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_print.c.o
[ 24%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_rand.c.o
[ 24%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_recp.c.o
[ 24%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_shift.c.o
[ 24%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_sqr.c.o
[ 24%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_sqrt.c.o
[ 25%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_word.c.o
[ 25%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bn/bn_x931p.c.o
[ 25%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/buffer/buf_err.c.o
[ 25%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/buffer/buf_str.c.o
[ 25%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/buffer/buffer.c.o
[ 25%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/camellia/cmll_cfb.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/camellia/cmll_ctr.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/camellia/cmll_ecb.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/camellia/cmll_misc.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/camellia/cmll_ofb.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cast/c_cfb64.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cast/c_ecb.c.o
[ 26%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cast/c_enc.c.o
[ 27%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cast/c_ofb64.c.o
[ 27%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cast/c_skey.c.o
[ 27%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/chacha/chacha.c.o
[ 27%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cmac/cm_ameth.c.o
[ 27%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cmac/cm_pmeth.c.o
[ 27%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cmac/cmac.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_asn1.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_att.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_cd.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_dd.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_enc.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_env.c.o
[ 28%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_err.c.o
[ 29%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_ess.c.o
[ 29%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_io.c.o
[ 29%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_kari.c.o
[ 29%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_lib.c.o
[ 29%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_pwri.c.o
[ 29%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_sd.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/cms/cms_smime.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/comp/c_rle.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/comp/c_zlib.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/comp/comp_err.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/comp/comp_lib.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_api.c.o
[ 30%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_def.c.o
[ 31%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_err.c.o
[ 31%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_lib.c.o
[ 31%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_mall.c.o
[ 31%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_mod.c.o
[ 31%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/conf/conf_sap.c.o
[ 31%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/curve25519/curve25519-generic.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/curve25519/curve25519.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/cbc_cksm.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/cbc_enc.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/cfb64ede.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/cfb64enc.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/cfb_enc.c.o
[ 32%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/des_enc.c.o
[ 33%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/ecb3_enc.c.o
[ 33%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/ecb_enc.c.o
[ 33%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/ede_cbcm_enc.c.o
[ 33%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/enc_read.c.o
[ 33%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/enc_writ.c.o
[ 33%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/fcrypt.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/fcrypt_b.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/ofb64enc.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/ofb64ede.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/ofb_enc.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/pcbc_enc.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/qud_cksm.c.o
[ 34%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/rand_key.c.o
[ 35%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/set_key.c.o
[ 35%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/str2key.c.o
[ 35%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/des/xcbc_enc.c.o
[ 35%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_ameth.c.o
[ 35%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_asn1.c.o
[ 35%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_check.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_depr.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_err.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_gen.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_key.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_lib.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_pmeth.c.o
[ 36%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dh/dh_prn.c.o
[ 37%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_ameth.c.o
[ 37%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_asn1.c.o
[ 37%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_depr.c.o
[ 37%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_err.c.o
[ 37%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_gen.c.o
[ 37%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_key.c.o
[ 38%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_lib.c.o
[ 38%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_meth.c.o
[ 38%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_ossl.c.o
[ 38%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_pmeth.c.o
[ 38%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_prn.c.o
[ 38%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_sign.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dsa/dsa_vrf.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dso/dso_dlfcn.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dso/dso_err.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dso/dso_lib.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dso/dso_null.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/dso/dso_openssl.c.o
[ 39%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec2_mult.c.o
[ 40%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec2_oct.c.o
[ 40%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec2_smpl.c.o
[ 40%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_ameth.c.o
[ 40%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_asn1.c.o
[ 40%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_check.c.o
[ 40%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_curve.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_cvt.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_err.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_key.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_kmeth.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_lib.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_mult.c.o
[ 41%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_oct.c.o
[ 42%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_pmeth.c.o
[ 42%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ec_print.c.o
[ 42%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/eck_prn.c.o
[ 42%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ecp_mont.c.o
[ 42%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ecp_nist.c.o
[ 42%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ecp_oct.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ec/ecp_smpl.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdh/ecdh_kdf.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdh/ech_err.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdh/ech_key.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdh/ech_lib.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdsa/ecs_asn1.c.o
[ 43%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdsa/ecs_err.c.o
[ 44%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdsa/ecs_lib.c.o
[ 44%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdsa/ecs_ossl.c.o
[ 44%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdsa/ecs_sign.c.o
[ 44%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ecdsa/ecs_vrf.c.o
[ 44%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_all.c.o
[ 44%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_cnf.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_ctrl.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_dyn.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_err.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_fat.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_init.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_lib.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_list.c.o
[ 45%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_pkey.c.o
[ 46%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_openssl.c.o
[ 46%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/eng_table.c.o
[ 46%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_asnmth.c.o
[ 46%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_cipher.c.o
[ 46%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_dh.c.o
[ 46%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_dsa.c.o
[ 47%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_digest.c.o
[ 47%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_ecdh.c.o
[ 47%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_ecdsa.c.o
[ 47%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_eckey.c.o
[ 47%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_pkmeth.c.o
[ 47%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_rand.c.o
[ 48%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_rsa.c.o
[ 48%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/engine/tb_store.c.o
[ 48%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/err/err.c.o
[ 48%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/err/err_all.c.o
[ 48%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/err/err_prn.c.o
[ 48%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/bio_b64.c.o
[ 49%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/bio_enc.c.o
[ 49%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/bio_md.c.o
[ 49%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/c_all.c.o
[ 49%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/digest.c.o
[ 49%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_aes.c.o
[ 49%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_aes_cbc_hmac_sha1.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_bf.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_camellia.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_cast.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_chacha.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_chacha20poly1305.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_des.c.o
[ 50%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_des3.c.o
[ 51%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_gost2814789.c.o
[ 51%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_null.c.o
[ 51%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_idea.c.o
[ 51%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_old.c.o
[ 51%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_rc2.c.o
[ 51%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_rc4.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_rc4_hmac_md5.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_sm4.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/e_xcbc_d.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/encode.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_aead.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_enc.c.o
[ 52%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_err.c.o
[ 53%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_key.c.o
[ 53%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_lib.c.o
[ 53%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_pbe.c.o
[ 53%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/evp_pkey.c.o
[ 53%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_dss.c.o
[ 53%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_dss1.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_ecdsa.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_gost2814789.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_gostr341194.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_md4.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_md5.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_md5_sha1.c.o
[ 54%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_null.c.o
[ 55%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_ripemd.c.o
[ 55%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_sha1.c.o
[ 55%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_sigver.c.o
[ 55%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_streebog.c.o
[ 55%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_sm3.c.o
[ 55%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/m_wp.c.o
[ 56%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/names.c.o
[ 56%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p5_crpt.c.o
[ 56%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p5_crpt2.c.o
[ 56%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_dec.c.o
[ 56%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_enc.c.o
[ 56%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_lib.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_seal.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_sign.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_open.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/p_verify.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/pmeth_fn.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/pmeth_gn.c.o
[ 57%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/evp/pmeth_lib.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost2814789.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost89_keywrap.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost89_params.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost89imit_ameth.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost89imit_pmeth.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost_asn1.c.o
[ 58%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gost_err.c.o
[ 59%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gostr341001.c.o
[ 59%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gostr341001_ameth.c.o
[ 59%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gostr341001_key.c.o
[ 59%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gostr341001_params.c.o
[ 59%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gostr341001_pmeth.c.o
[ 59%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/gostr341194.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/gost/streebog.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/hkdf/hkdf.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/hmac/hm_ameth.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/hmac/hm_pmeth.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/hmac/hmac.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/idea/i_cbc.c.o
[ 60%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/idea/i_cfb64.c.o
[ 61%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/idea/i_ecb.c.o
[ 61%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/idea/i_ofb64.c.o
[ 61%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/idea/i_skey.c.o
[ 61%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/lhash/lh_stats.c.o
[ 61%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/lhash/lhash.c.o
[ 61%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/md4/md4_dgst.c.o
[ 62%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/md4/md4_one.c.o
[ 62%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/md5/md5_dgst.c.o
[ 62%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/md5/md5_one.c.o
[ 62%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/cbc128.c.o
[ 62%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/ccm128.c.o
[ 62%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/cfb128.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/ctr128.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/cts128.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/gcm128.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/ofb128.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/modes/xts128.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/objects/o_names.c.o
[ 63%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/objects/obj_dat.c.o
[ 64%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/objects/obj_err.c.o
[ 64%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/objects/obj_lib.c.o
[ 64%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/objects/obj_xref.c.o
[ 64%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_asn.c.o
[ 64%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_cl.c.o
[ 64%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_err.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_ext.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_ht.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_lib.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_prn.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_srv.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ocsp/ocsp_vfy.c.o
[ 65%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_all.c.o
[ 66%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_err.c.o
[ 66%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_info.c.o
[ 66%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_lib.c.o
[ 66%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_oth.c.o
[ 66%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_pk8.c.o
[ 66%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_pkey.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_seal.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_sign.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_x509.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pem_xaux.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pem/pvkfmt.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_add.c.o
[ 67%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_asn.c.o
[ 68%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_attr.c.o
[ 68%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_crpt.c.o
[ 68%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_crt.c.o
[ 68%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_decr.c.o
[ 68%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_init.c.o
[ 68%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_key.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_kiss.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_mutl.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_npas.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_p8d.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_p8e.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/p12_utl.c.o
[ 69%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs12/pk12err.c.o
[ 70%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/bio_pk7.c.o
[ 70%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pk7_asn1.c.o
[ 70%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pk7_attr.c.o
[ 70%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pk7_doit.c.o
[ 70%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pk7_lib.c.o
[ 70%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pk7_mime.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pk7_smime.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/pkcs7/pkcs7err.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/poly1305/poly1305.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rand/rand_err.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rand/rand_lib.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rand/randfile.c.o
[ 71%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc2/rc2_cbc.c.o
[ 72%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc2/rc2_ecb.c.o
[ 72%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc2/rc2_skey.c.o
[ 72%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc2/rc2cfb64.c.o
[ 72%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rc2/rc2ofb64.c.o
[ 72%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ripemd/rmd_dgst.c.o
[ 72%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ripemd/rmd_one.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_ameth.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_asn1.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_chk.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_crpt.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_depr.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_eay.c.o
[ 73%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_err.c.o
[ 74%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_gen.c.o
[ 74%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_lib.c.o
[ 74%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_meth.c.o
[ 74%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_none.c.o
[ 74%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_oaep.c.o
[ 74%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_pk1.c.o
[ 75%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_pmeth.c.o
[ 75%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_prn.c.o
[ 75%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_pss.c.o
[ 75%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_saos.c.o
[ 75%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_sign.c.o
[ 75%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/rsa/rsa_x931.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha1_one.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha1dgst.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha256.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sha/sha512.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sm3/sm3.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/sm4/sm4.c.o
[ 76%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/stack/stack.c.o
[ 77%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_asn1.c.o
[ 77%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_conf.c.o
[ 77%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_err.c.o
[ 77%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_lib.c.o
[ 77%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_req_print.c.o
[ 77%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_req_utils.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_rsp_print.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_rsp_sign.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_rsp_utils.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_rsp_verify.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ts/ts_verify_ctx.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/txt_db/txt_db.c.o
[ 78%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ui/ui_err.c.o
[ 79%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ui/ui_lib.c.o
[ 79%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ui/ui_util.c.o
[ 79%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/whrlpool/wp_dgst.c.o
[ 79%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/by_dir.c.o
[ 79%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/by_file.c.o
[ 79%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/by_mem.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_att.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_cmp.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_d2.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_def.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_err.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_ext.c.o
[ 80%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_lu.c.o
[ 81%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_obj.c.o
[ 81%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_r2x.c.o
[ 81%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_req.c.o
[ 81%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_set.c.o
[ 81%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_trs.c.o
[ 81%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_txt.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_v3.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_vfy.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509_vpm.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509cset.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509name.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509rset.c.o
[ 82%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509spki.c.o
[ 83%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x509type.c.o
[ 83%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509/x_all.c.o
[ 83%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/pcy_cache.c.o
[ 83%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/pcy_data.c.o
[ 83%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/pcy_lib.c.o
[ 83%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/pcy_map.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/pcy_node.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/pcy_tree.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_akeya.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_akey.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_alt.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_bcons.c.o
[ 84%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_bitst.c.o
[ 85%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_conf.c.o
[ 85%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_cpols.c.o
[ 85%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_crld.c.o
[ 85%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_enum.c.o
[ 85%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_extku.c.o
[ 85%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_genn.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_ia5.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_info.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_int.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_lib.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_ncons.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_ocsp.c.o
[ 86%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_pci.c.o
[ 87%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_pcia.c.o
[ 87%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_pcons.c.o
[ 87%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_pku.c.o
[ 87%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_pmaps.c.o
[ 87%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_prn.c.o
[ 87%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_purp.c.o
[ 88%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_skey.c.o
[ 88%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_sxnet.c.o
[ 88%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3_utl.c.o
[ 88%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/x509v3/v3err.c.o
[ 88%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/crypto_lock.c.o
[ 88%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/b_posix.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/bio/bss_log.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/ui/ui_openssl.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/compat/freezero.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/compat/reallocarray.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/compat/recallocarray.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/compat/syslog_r.c.o
[ 89%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/compat/explicit_bzero.c.o
[ 90%] Building C object source_subfolder/crypto/CMakeFiles/crypto.dir/compat/timingsafe_memcmp.c.o
[ 90%] Linking C static library ../../lib/libcrypto.a
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: ../../lib/libcrypto.a(cms_cd.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: ../../lib/libcrypto.a(cms_cd.c.o) has no symbols
[ 90%] Built target crypto
Scanning dependencies of target ssl
[ 90%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/bs_ber.c.o
[ 90%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/bs_cbb.c.o
[ 90%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/bio_ssl.c.o
[ 90%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/bs_cbs.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/d1_both.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/d1_clnt.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/d1_lib.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/d1_pkt.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/d1_srtp.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/d1_srvr.c.o
[ 91%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/pqueue.c.o
[ 92%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/s3_cbc.c.o
[ 92%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/s3_lib.c.o
[ 92%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_algs.c.o
[ 92%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_asn1.c.o
[ 92%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_both.c.o
[ 92%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_cert.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_ciph.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_ciphers.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_clnt.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_err.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_init.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_kex.c.o
[ 93%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_lib.c.o
[ 94%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_methods.c.o
[ 94%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_packet.c.o
[ 94%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_pkt.c.o
[ 94%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_rsa.c.o
[ 94%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_sess.c.o
[ 94%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_sigalgs.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_srvr.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_stat.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_tlsext.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_transcript.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_txt.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/ssl_versions.c.o
[ 95%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/t1_enc.c.o
[ 96%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/t1_lib.c.o
[ 96%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_buffer.c.o
[ 96%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_client.c.o
[ 96%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_error.c.o
[ 96%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_handshake.c.o
[ 96%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_handshake_msg.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_key_schedule.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_key_share.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_legacy.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_lib.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_record.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_record_layer.c.o
[ 97%] Building C object source_subfolder/ssl/CMakeFiles/ssl.dir/tls13_server.c.o
[ 98%] Linking C static library ../../lib/libssl.a
[ 98%] Built target ssl
Scanning dependencies of target tls
[ 98%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls.c.o
[ 98%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_client.c.o
[ 98%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_config.c.o
[ 98%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_bio_cb.c.o
[ 98%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_conninfo.c.o
[ 99%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_keypair.c.o
[ 99%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_server.c.o
[ 99%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_ocsp.c.o
[ 99%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_peer.c.o
[ 99%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_util.c.o
[ 99%] Building C object source_subfolder/tls/CMakeFiles/tls.dir/tls_verify.c.o
[100%] Linking C static library ../../lib/libtls.a
[100%] Built target tls
libressl/3.2.0: Package 'f74366f76f700cc6e991285892ad7a23c30e6d47' built
libressl/3.2.0: Build folder /Users/n1/.conan/data/libressl/3.2.0/_/_/build/f74366f76f700cc6e991285892ad7a23c30e6d47
libressl/3.2.0: Generated conaninfo.txt
libressl/3.2.0: Generated conanbuildinfo.txt
libressl/3.2.0: Generating the package
libressl/3.2.0: Package folder /Users/n1/.conan/data/libressl/3.2.0/_/_/package/f74366f76f700cc6e991285892ad7a23c30e6d47
libressl/3.2.0: Calling package()
libressl/3.2.0 package(): Packaged 1 file: COPYING
libressl/3.2.0 package(): Packaged 73 '.h' files
libressl/3.2.0 package(): Packaged 3 '.a' files: libtls.a, libcrypto.a, libssl.a
libressl/3.2.0: Package 'f74366f76f700cc6e991285892ad7a23c30e6d47' created
libressl/3.2.0: Created package revision 141e4a8120c5093959dfe3791afaec9e
mongo-c-driver/1.17.2@dev/testing: WARN: Build folder is dirty, removing it: /Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/build/f03349b8a70d68ed7d068c77dae611fdcfdbcf9e
mongo-c-driver/1.17.2@dev/testing: Configuring sources in /Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/source
Downloading mongo-c-driver-1.17.2.tar.gz completed [6922.22k]                                

mongo-c-driver/1.17.2@dev/testing: Building your package in /Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/build/f03349b8a70d68ed7d068c77dae611fdcfdbcf9e
mongo-c-driver/1.17.2@dev/testing: Generator cmake created conanbuildinfo.cmake
mongo-c-driver/1.17.2@dev/testing: Calling build()
-- The C compiler identification is ;AppleClang 11.0.0.11000033
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
file VERSION_CURRENT contained BUILD_VERSION 1.17.2
-- Build and install static libraries
  -- Using bundled libbson
libbson version (from VERSION_CURRENT file): 1.17.2
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Searching 16 bit integer - Using unsigned short
-- Check if the system is big endian - little endian
-- Looking for snprintf
-- Looking for snprintf - found
-- Looking for reallocf
-- Looking for reallocf - found
-- Performing Test BSON_HAVE_TIMESPEC
-- Performing Test BSON_HAVE_TIMESPEC - Success
--     struct timespec found
-- Looking for gmtime_r
-- Looking for gmtime_r - found
-- Looking for rand_r
-- Looking for rand_r - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for strlcpy
-- Looking for strlcpy - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for strnlen
-- Looking for strnlen - found
-- Looking for stdbool.h
-- Looking for stdbool.h - found
-- Looking for SYS_gettid
-- Looking for SYS_gettid - found
-- Looking for syscall
-- Looking for syscall - found
-- Performing Test HAVE_ATOMIC_32_ADD_AND_FETCH
-- Performing Test HAVE_ATOMIC_32_ADD_AND_FETCH - Success
-- Performing Test HAVE_ATOMIC_64_ADD_AND_FETCH
-- Performing Test HAVE_ATOMIC_64_ADD_AND_FETCH - Success
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
Adding -fPIC to compilation of bson_static components
libmongoc version (from VERSION_CURRENT file): 1.17.2
-- Enabling zlib compression (bundled)
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include file stdarg.h
-- Looking for include file stdarg.h - found
-- Searching for compression library zstd
-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2") 
-- Checking for module 'libzstd'
--   Found libzstd, version 1.4.5
--   Found zstd version 1.4.5 in /usr/local/Cellar/zstd/1.4.5//include
--   USE LIBRESSL include: /Users/n1/.conan/data/libressl/3.2.0/_/_/package/f74366f76f700cc6e991285892ad7a23c30e6d47/include!
--   USE LIBRESSL library: /Users/n1/.conan/data/libressl/3.2.0/_/_/package/f74366f76f700cc6e991285892ad7a23c30e6d47/lib!
-- Searching for sasl/sasl.h
--   Found in /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include
-- Searching for libsasl2
--   Found /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib/libsasl2.tbd
-- Check size of socklen_t
-- Check size of socklen_t - done
-- Looking for res_nsearch
-- Looking for res_nsearch - found
-- Looking for res_ndestroy
-- Looking for res_ndestroy - found
-- Looking for sched_getcpu
-- Looking for sched_getcpu - not found
-- Detected parameters: accept (int, struct sockaddr *, socklen_t *)
-- Searching for compression library header snappy-c.h
--   Found in /usr/local/include
-- Searching for libsnappy
--   Found /usr/local/lib/libsnappy.dylib
-- No ICU library found, SASLPrep disabled for SCRAM-SHA-256 authentication.
-- If ICU is installed in a non-standard directory, define ICU_ROOT as the ICU installation path.
Searching for libmongocrypt
-- libmongocrypt not found. Configuring without Client-Side Field Level Encryption support.
-- Performing Test MONGOC_HAVE_SS_FAMILY
-- Performing Test MONGOC_HAVE_SS_FAMILY - Success
-- Compiling against LibreSSL
-- Compiling against Cyrus SASL
Adding -fPIC to compilation of mongoc_static components
-- Building with MONGODB-AWS auth support
-- Build files generated for:
--      build system: Unix Makefiles
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP
    CONAN_CMAKE_POSITION_INDEPENDENT_CODE
    CONAN_COMPILER
    CONAN_COMPILER_VERSION
    CONAN_CXX_FLAGS
    CONAN_C_FLAGS
    CONAN_EXPORTED
    CONAN_IN_LOCAL_CACHE
    CONAN_SHARED_LINKER_FLAGS


-- Build files have been written to: /Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/build/f03349b8a70d68ed7d068c77dae611fdcfdbcf9e/build_subfolder
Scanning dependencies of target bson_static
Scanning dependencies of target bson_shared
[  0%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bcon.c.o
[  0%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bcon.c.o
[  0%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson.c.o
[  0%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-atomic.c.o
[  1%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-clock.c.o
[  1%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-context.c.o
[  1%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-decimal128.c.o
[  2%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson.c.o
[  3%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-error.c.o
[  3%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-atomic.c.o
[  3%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-iso8601.c.o
[  3%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-clock.c.o
[  3%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-iter.c.o
[  4%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-context.c.o
[  5%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-json.c.o
[  5%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-decimal128.c.o
[  5%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-keys.c.o
[  5%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-error.c.o
[  6%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-iso8601.c.o
[  6%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-iter.c.o
[  6%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-json.c.o
[  6%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-md5.c.o
[  7%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-keys.c.o
[  7%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-md5.c.o
[  8%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-memory.c.o
[  8%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-oid.c.o
[  8%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-reader.c.o
[  9%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-string.c.o
[  9%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-memory.c.o
[ 10%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-oid.c.o
[ 10%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-timegm.c.o
[ 10%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-reader.c.o
[ 10%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-utf8.c.o
[ 11%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-value.c.o
[ 11%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-string.c.o
[ 12%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-timegm.c.o
[ 12%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-utf8.c.o
[ 12%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-version-functions.c.o
[ 12%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/bson/bson-writer.c.o
[ 12%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-value.c.o
[ 13%] Building C object src/libbson/CMakeFiles/bson_shared.dir/src/jsonsl/jsonsl.c.o
[ 13%] Building C object src/libbson/CMakeFiles/bson_shared.dir/__/common/common-b64.c.o
[ 14%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-version-functions.c.o
[ 14%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/bson/bson-writer.c.o
[ 14%] Building C object src/libbson/CMakeFiles/bson_shared.dir/__/common/common-md5.c.o
[ 14%] Building C object src/libbson/CMakeFiles/bson_static.dir/src/jsonsl/jsonsl.c.o
[ 15%] Building C object src/libbson/CMakeFiles/bson_static.dir/__/common/common-b64.c.o
[ 15%] Building C object src/libbson/CMakeFiles/bson_static.dir/__/common/common-md5.c.o
[ 16%] Building C object src/libbson/CMakeFiles/bson_shared.dir/__/common/common-thread.c.o
[ 16%] Building C object src/libbson/CMakeFiles/bson_static.dir/__/common/common-thread.c.o
[ 16%] Linking C shared library libbson-1.0.dylib
[ 16%] Built target bson_shared
Scanning dependencies of target mongoc_shared
[ 16%] Linking C static library libbson-static-1.0.a
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libbson-static-1.0.a(bson-atomic.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libbson-static-1.0.a(bson-atomic.c.o) has no symbols
[ 16%] Built target bson_static
Scanning dependencies of target mongoc_static
[ 17%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/crc32.c.o
[ 17%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/deflate.c.o
[ 17%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/adler32.c.o
[ 17%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/adler32.c.o
[ 18%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/infback.c.o
[ 19%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/crc32.c.o
[ 19%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/deflate.c.o
[ 19%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/inffast.c.o
[ 19%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/infback.c.o
[ 20%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/inffast.c.o
[ 20%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/inflate.c.o
[ 20%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/inflate.c.o
[ 20%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/inftrees.c.o
[ 21%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/inftrees.c.o
[ 21%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/trees.c.o
[ 22%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/trees.c.o
[ 22%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/zutil.c.o
[ 22%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/zutil.c.o
[ 23%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/compress.c.o
[ 23%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/compress.c.o
[ 24%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/uncompr.c.o
[ 24%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/gzclose.c.o
[ 24%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/uncompr.c.o
[ 25%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/gzclose.c.o
[ 25%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/gzlib.c.o
[ 25%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/gzlib.c.o
[ 26%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/gzread.c.o
[ 26%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/zlib-1.2.11/gzwrite.c.o
[ 26%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/gzread.c.o
[ 26%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-aggregate.c.o
[ 27%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-apm.c.o
[ 27%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/zlib-1.2.11/gzwrite.c.o
[ 28%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-aggregate.c.o
[ 28%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-apm.c.o
[ 28%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-array.c.o
[ 28%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-array.c.o
[ 28%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-async.c.o
[ 29%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-async-cmd.c.o
[ 29%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-buffer.c.o
[ 30%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-async.c.o
[ 30%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-async-cmd.c.o
[ 30%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-buffer.c.o
[ 30%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-bulk-operation.c.o
[ 31%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-change-stream.c.o
[ 31%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-client.c.o
[ 32%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-bulk-operation.c.o
[ 32%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-change-stream.c.o
[ 32%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-client.c.o
[ 32%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-client-pool.c.o
[ 32%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-client-side-encryption.c.o
[ 33%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-client-pool.c.o
[ 33%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-client-side-encryption.c.o
[ 34%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cluster.c.o
[ 34%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cluster-aws.c.o
[ 34%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cluster-sasl.c.o
[ 34%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cluster.c.o
[ 35%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cluster-aws.c.o
[ 35%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cluster-sasl.c.o
[ 36%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-collection.c.o
[ 36%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-compression.c.o
[ 36%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-collection.c.o
[ 37%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-compression.c.o
[ 37%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-counters.c.o
[ 38%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-crypt.c.o
[ 38%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-counters.c.o
[ 38%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-crypt.c.o
[ 38%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-array.c.o
[ 38%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor.c.o
[ 39%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-array.c.o
[ 40%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-cmd.c.o
[ 40%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor.c.o
[ 40%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-change-stream.c.o
[ 40%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-cmd.c.o
[ 41%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-change-stream.c.o
[ 41%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-cmd-deprecated.c.o
[ 41%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-cmd-deprecated.c.o
[ 42%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-find.c.o
[ 42%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-find.c.o
[ 42%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-find-cmd.c.o
[ 42%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-find-opquery.c.o
[ 43%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cursor-legacy.c.o
[ 44%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-find-cmd.c.o
[ 44%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-find-opquery.c.o
[ 44%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cursor-legacy.c.o
[ 44%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-database.c.o
[ 44%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-error.c.o
[ 45%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-database.c.o
[ 46%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-find-and-modify.c.o
[ 46%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-error.c.o
[ 46%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-init.c.o
/Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/source/source_subfolder/src/libmongoc/src/mongoc/mongoc-init.c:47:9: warning: Configure the driver with
      ENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF. Automatic cleanup is deprecated and will be removed in version 2.0. [-W#pragma-messages]
#pragma message( \
        ^
[ 46%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-find-and-modify.c.o
1 warning generated.
[ 46%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-gridfs.c.o
[ 47%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-gridfs-bucket.c.o
[ 47%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-gridfs-bucket-file.c.o
[ 48%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-init.c.o
[ 48%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-gridfs.c.o
[ 48%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-gridfs-file.c.o
[ 48%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-gridfs-bucket.c.o
/Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/source/source_subfolder/src/libmongoc/src/mongoc/mongoc-init.c:47:9: warning: Configure the driver with
      ENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF. Automatic cleanup is deprecated and will be removed in version 2.0. [-W#pragma-messages]
#pragma message( \
        ^
1 warning generated.
[ 49%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-gridfs-file-list.c.o
[ 49%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-gridfs-file-page.c.o
[ 50%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-gridfs-bucket-file.c.o
[ 50%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-handshake.c.o
[ 50%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-gridfs-file.c.o
[ 50%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-gridfs-file-list.c.o
[ 51%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-host-list.c.o
[ 51%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-http.c.o
[ 51%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-gridfs-file-page.c.o
[ 51%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-index.c.o
[ 52%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-handshake.c.o
[ 52%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-host-list.c.o
[ 53%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-interrupt.c.o
[ 53%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-list.c.o
[ 53%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-linux-distro-scanner.c.o
[ 54%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-log.c.o
[ 54%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-http.c.o
[ 54%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-matcher.c.o
[ 55%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-index.c.o
[ 55%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-matcher-op.c.o
[ 56%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-memcmp.c.o
[ 56%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-interrupt.c.o
[ 56%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cmd.c.o
[ 56%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-list.c.o
[ 56%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-opts-helpers.c.o
[ 57%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-linux-distro-scanner.c.o
[ 57%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-log.c.o
[ 57%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-matcher.c.o
[ 58%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-matcher-op.c.o
[ 59%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-opts.c.o
[ 59%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-queue.c.o
[ 59%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-memcmp.c.o
[ 59%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-read-concern.c.o
[ 59%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cmd.c.o
[ 60%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-read-prefs.c.o
[ 61%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-opts-helpers.c.o
[ 61%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-rpc.c.o
[ 61%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-opts.c.o
[ 61%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-queue.c.o
[ 61%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-server-description.c.o
[ 62%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-read-concern.c.o
[ 63%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-server-stream.c.o
[ 63%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-read-prefs.c.o
[ 63%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-rpc.c.o
[ 63%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-client-session.c.o
[ 64%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-server-description.c.o
[ 64%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-server-monitor.c.o
[ 64%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-server-stream.c.o
[ 65%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-set.c.o
[ 65%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-client-session.c.o
[ 65%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-socket.c.o
[ 65%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-buffered.c.o
[ 66%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-server-monitor.c.o
[ 66%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-set.c.o
[ 66%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream.c.o
[ 67%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-file.c.o
[ 67%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-gridfs.c.o
[ 67%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-socket.c.o
[ 68%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-buffered.c.o
[ 68%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream.c.o
[ 68%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-gridfs-download.c.o
[ 69%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-gridfs-upload.c.o
[ 69%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-file.c.o
[ 69%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-socket.c.o
[ 69%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-topology.c.o
[ 70%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-topology-background-monitoring.c.o
[ 71%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-gridfs.c.o
[ 71%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-gridfs-download.c.o
[ 71%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-gridfs-upload.c.o
[ 71%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-topology-description.c.o
[ 71%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-topology-description-apm.c.o
[ 72%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-socket.c.o
[ 72%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-topology.c.o
[ 72%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-topology-background-monitoring.c.o
[ 73%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-topology-scanner.c.o
[ 73%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-uri.c.o
[ 74%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-topology-description.c.o
[ 74%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-util.c.o
[ 74%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-topology-description-apm.c.o
[ 74%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-topology-scanner.c.o
[ 75%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-version-functions.c.o
[ 75%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-write-command.c.o
[ 76%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-uri.c.o
[ 76%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-write-command-legacy.c.o
[ 76%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-util.c.o
[ 76%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-version-functions.c.o
[ 77%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-write-concern.c.o
[ 77%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/common/common-b64.c.o
[ 77%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/common/common-md5.c.o
[ 78%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-write-command.c.o
[ 78%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-write-command-legacy.c.o
[ 79%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/common/common-thread.c.o
[ 79%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-crypto.c.o
[ 79%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-write-concern.c.o
[ 79%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-scram.c.o
[ 80%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/common/common-b64.c.o
[ 81%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-tls.c.o
[ 81%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-ssl.c.o
[ 81%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/common/common-md5.c.o
[ 81%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/common/common-thread.c.o
[ 81%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-crypto-openssl.c.o
[ 82%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-crypto.c.o
[ 83%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-rand-openssl.c.o
[ 83%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-stream-tls-libressl.c.o
[ 83%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-libressl.c.o
[ 83%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-scram.c.o
[ 83%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-tls.c.o
[ 84%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-sasl.c.o
[ 85%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-ssl.c.o
[ 85%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cluster-cyrus.c.o
[ 85%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-crypto-openssl.c.o
[ 85%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/src/mongoc/mongoc-cyrus.c.o
[ 86%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/hexlify.c.o
[ 86%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-rand-openssl.c.o
[ 86%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_b64.c.o
[ 86%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-stream-tls-libressl.c.o
[ 87%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-libressl.c.o
[ 87%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_caller_identity_request.c.o
[ 87%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-sasl.c.o
[ 87%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cluster-cyrus.c.o
[ 88%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_crypto_apple.c.o
[ 88%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_crypto_libcrypto.c.o
[ 88%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_crypto_none.c.o
[ 89%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_crypto_windows.c.o
[ 89%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_decrypt_request.c.o
[ 90%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-cyrus.c.o
[ 90%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/hexlify.c.o
[ 90%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_b64.c.o
[ 90%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_encrypt_request.c.o
[ 91%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_kv_list.c.o
[ 91%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_message.c.o
[ 92%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_caller_identity_request.c.o
[ 92%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_crypto_apple.c.o
[ 92%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_port.c.o
[ 93%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_request.c.o
[ 93%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_crypto_libcrypto.c.o
[ 93%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_request_opt.c.o
[ 93%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_request_str.c.o
[ 94%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_crypto_none.c.o
[ 95%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_response.c.o
[ 95%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_crypto_windows.c.o
[ 95%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/kms_response_parser.c.o
[ 95%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_decrypt_request.c.o
[ 96%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_encrypt_request.c.o
[ 96%] Building C object src/libmongoc/CMakeFiles/mongoc_static.dir/__/kms-message/src/sort.c.o
[ 96%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_kv_list.c.o
[ 96%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_message.c.o
[ 97%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_port.c.o
[ 97%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_request.c.o
[ 97%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_request_opt.c.o
[ 98%] Linking C static library libmongoc-static-1.0.a
[ 99%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_request_str.c.o
[ 99%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_response.c.o
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(mongoc-crypt.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(mongoc-linux-distro-scanner.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_crypto_apple.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_crypto_none.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_crypto_windows.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_port.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(mongoc-crypt.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(mongoc-linux-distro-scanner.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_crypto_apple.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_crypto_none.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_crypto_windows.c.o) has no symbols
/Library/Developer/CommandLineTools/usr/bin/ranlib: file: libmongoc-static-1.0.a(kms_port.c.o) has no symbols
[ 99%] Built target mongoc_static
[ 99%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/kms_response_parser.c.o
[100%] Building C object src/libmongoc/CMakeFiles/mongoc_shared.dir/__/kms-message/src/sort.c.o
[100%] Linking C shared library libmongoc-1.0.dylib
Undefined symbols for architecture x86_64:
  "_OPENSSL_init_ssl", referenced from:
      _tls_do_init in libtls.a(tls.c.o)
  "_PEM_read_bio_SSL_SESSION", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
  "_PEM_write_bio_SSL_SESSION", referenced from:
      _tls_handshake_client in libtls.a(tls_client.c.o)
  "_SSL_CIPHER_get_bits", referenced from:
      _tls_conninfo_populate in libtls.a(tls_conninfo.c.o)
  "_SSL_CIPHER_get_name", referenced from:
      _tls_conninfo_populate in libtls.a(tls_conninfo.c.o)
  "_SSL_CTX_callback_ctrl", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_check_private_key", referenced from:
      _tls_configure_ssl_keypair in libtls.a(tls.c.o)
  "_SSL_CTX_ctrl", referenced from:
      _tls_configure_ssl in libtls.a(tls.c.o)
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_free", referenced from:
      _tls_sni_ctx_free in libtls.a(tls.c.o)
      _tls_reset in libtls.a(tls.c.o)
      _tls_config_set_ciphers in libtls.a(tls_config.c.o)
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_get_cert_store", referenced from:
      _tls_configure_ssl_verify in libtls.a(tls.c.o)
      _tls_ocsp_process_response_internal in libtls.a(tls_ocsp.c.o)
  "_SSL_CTX_load_verify_locations", referenced from:
      _tls_configure_ssl_verify in libtls.a(tls.c.o)
  "_SSL_CTX_load_verify_mem", referenced from:
      _tls_configure_ssl_verify in libtls.a(tls.c.o)
  "_SSL_CTX_new", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_config_set_ciphers in libtls.a(tls_config.c.o)
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_set1_groups", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_set_alpn_protos", referenced from:
      _tls_configure_ssl in libtls.a(tls.c.o)
  "_SSL_CTX_set_alpn_select_cb", referenced from:
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_set_cert_verify_callback", referenced from:
      _tls_configure_ssl_verify in libtls.a(tls.c.o)
  "_SSL_CTX_set_cipher_list", referenced from:
      _tls_configure_ssl in libtls.a(tls.c.o)
      _tls_config_set_ciphers in libtls.a(tls_config.c.o)
  "_SSL_CTX_set_session_id_context", referenced from:
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_set_timeout", referenced from:
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
  "_SSL_CTX_set_verify", referenced from:
      _tls_configure_ssl_verify in libtls.a(tls.c.o)
  "_SSL_CTX_set_verify_depth", referenced from:
      _tls_configure_ssl_verify in libtls.a(tls.c.o)
  "_SSL_CTX_use_PrivateKey", referenced from:
      _tls_configure_ssl_keypair in libtls.a(tls.c.o)
  "_SSL_CTX_use_certificate_chain_mem", referenced from:
      _tls_configure_ssl_keypair in libtls.a(tls.c.o)
  "_SSL_SESSION_free", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_handshake_client in libtls.a(tls_client.c.o)
  "_SSL_accept", referenced from:
      _tls_handshake_server in libtls.a(tls_server.c.o)
  "_SSL_connect", referenced from:
      _tls_handshake_client in libtls.a(tls_client.c.o)
  "_SSL_ctrl", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_conninfo_populate in libtls.a(tls_conninfo.c.o)
      _tls_ocsp_verify_cb in libtls.a(tls_ocsp.c.o)
      _tls_ocsp_stapling_cb in libtls.a(tls_ocsp.c.o)
  "_SSL_free", referenced from:
      _tls_reset in libtls.a(tls.c.o)
  "_SSL_get0_alpn_selected", referenced from:
      _tls_conninfo_populate in libtls.a(tls_conninfo.c.o)
  "_SSL_get1_session", referenced from:
      _tls_handshake_client in libtls.a(tls_client.c.o)
  "_SSL_get_current_cipher", referenced from:
      _tls_conninfo_populate in libtls.a(tls_conninfo.c.o)
  "_SSL_get_error", referenced from:
      _tls_ssl_error in libtls.a(tls.c.o)
  "_SSL_get_ex_data", referenced from:
      _tls_servername_cb in libtls.a(tls_server.c.o)
      _tls_server_ticket_cb in libtls.a(tls_server.c.o)
      _tls_ocsp_verify_cb in libtls.a(tls_ocsp.c.o)
      _tls_ocsp_stapling_cb in libtls.a(tls_ocsp.c.o)
  "_SSL_get_peer_cert_chain", referenced from:
      _tls_handshake in libtls.a(tls.c.o)
      _tls_ocsp_setup_from_peer in libtls.a(tls_ocsp.c.o)
  "_SSL_get_peer_certificate", referenced from:
      _tls_handshake_client in libtls.a(tls_client.c.o)
      _tls_handshake in libtls.a(tls.c.o)
      _tls_ocsp_setup_from_peer in libtls.a(tls_ocsp.c.o)
  "_SSL_get_servername", referenced from:
      _tls_servername_cb in libtls.a(tls_server.c.o)
  "_SSL_get_version", referenced from:
      _tls_conninfo_populate in libtls.a(tls_conninfo.c.o)
  "_SSL_new", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_accept_common in libtls.a(tls_server.c.o)
  "_SSL_read", referenced from:
      _tls_read in libtls.a(tls.c.o)
  "_SSL_select_next_proto", referenced from:
      _tls_server_alpn_cb in libtls.a(tls_server.c.o)
  "_SSL_set_SSL_CTX", referenced from:
      _tls_servername_cb in libtls.a(tls_server.c.o)
  "_SSL_set_bio", referenced from:
      _tls_set_cbs in libtls.a(tls_bio_cb.c.o)
  "_SSL_set_ex_data", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
      _tls_accept_common in libtls.a(tls_server.c.o)
  "_SSL_set_rfd", referenced from:
      _tls_connect_servername in libtls.a(tls_client.c.o)
      _tls_connect_socket in libtls.a(tls_client.c.o)
      _tls_connect_fds in libtls.a(tls_client.c.o)
      _tls_accept_socket in libtls.a(tls_server.c.o)
      _tls_accept_fds in libtls.a(tls_server.c.o)
  "_SSL_set_session", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
  "_SSL_set_wfd", referenced from:
      _tls_connect_servername in libtls.a(tls_client.c.o)
      _tls_connect_socket in libtls.a(tls_client.c.o)
      _tls_connect_fds in libtls.a(tls_client.c.o)
      _tls_accept_socket in libtls.a(tls_server.c.o)
      _tls_accept_fds in libtls.a(tls_server.c.o)
  "_SSL_shutdown", referenced from:
      _tls_close in libtls.a(tls.c.o)
  "_SSL_write", referenced from:
      _tls_write in libtls.a(tls.c.o)
  "_SSLv23_client_method", referenced from:
      _tls_connect_common in libtls.a(tls_client.c.o)
  "_SSLv23_method", referenced from:
      _tls_config_set_ciphers in libtls.a(tls_config.c.o)
  "_SSLv23_server_method", referenced from:
      _tls_configure_server_ssl in libtls.a(tls_server.c.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
gmake[2]: *** [src/libmongoc/CMakeFiles/mongoc_shared.dir/build.make:1982: src/libmongoc/libmongoc-1.0.0.0.0.dylib] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:732: src/libmongoc/CMakeFiles/mongoc_shared.dir/all] Error 2
gmake: *** [Makefile:171: all] Error 2
mongo-c-driver/1.17.2@dev/testing: 
mongo-c-driver/1.17.2@dev/testing: ERROR: Package 'f03349b8a70d68ed7d068c77dae611fdcfdbcf9e' build failed
mongo-c-driver/1.17.2@dev/testing: WARN: Build folder /Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/build/f03349b8a70d68ed7d068c77dae611fdcfdbcf9e
ERROR: mongo-c-driver/1.17.2@dev/testing: Error in build() method, line 49
        cmake.build()
        ConanException: Error 2 while executing cmake --build '/Users/n1/.conan/data/mongo-c-driver/1.17.2/dev/testing/build/f03349b8a70d68ed7d068c77dae611fdcfdbcf9e/build_subfolder' '--' '-j4'

</details>

Reply:
addressed in n-bes#1 + requires #2933 to be merged